quickwit/quickwit-cli/src/index.rs

```diff
@@ -121,6 +121,16 @@ pub fn build_index_command<'a>() -> Command<'a> {
                         .display_order(1),
                     arg!(--"input-path" <INPUT_PATH> "Location of the input file.")
                         .required(false),
+                    Arg::new("wait")
+                        .long("wait")
+                        .short('w')
+                        .help("Wait for a document to be commited before exiting")
```

Comment:
Maybe "Wait for all documents to be committed and available for search before exiting"?